test/api.js

```diff
@@ -565,66 +554,6 @@ function generateTests(prefix, apiCreator) {
 		return api.run([path.join(__dirname, 'fixture/immediate-3-exit.js')]);
 	});
 
-	test(prefix + 'testing nonexistent files causes an AvaError to be emitted', function (t) {
-		t.plan(2);
-
-		var api = apiCreator();
-
-		api.on('test-run', function (runStatus) {
-			runStatus.on('error', function (err) {
-				t.is(err.name, 'AvaError');
-				t.match(err.message, /Couldn't find any files to test/);
-			});
-		});
-
-		return api.run([path.join(__dirname, 'fixture/broken.js')]);
-	});
-
-	test(prefix + 'test file in node_modules is ignored', function (t) {
```

Comment:
Each behavior remains verified, because there are all those should-be-ignored directories in the fixture, that `AvaFiles` tests use. If at least one directory was not ignored, test would fail.